frontend/src/components/organisms/MatchesList/MatchesList.tsx

```diff
@@ -30,6 +37,47 @@ export const MatchesList = ({tournamentId}: MatchesListProps) => {
          });
    }, [tournamentId]);
 
+   useEffect(() => {
+      if (!matchesListItems) return;
+
+      async function setMatchesInformationListDtoState(): Promise<void> {
+         if (!matchesListItems) return;
+         const returnedMatchesInformationListDto = await Promise.all(matchesListItems
+            .map(async (matchListItem) => {
+                  const matchId = tournamentId + matchListItem.matchNumber;
+                  return await getMatchInformationDto(matchId)
+               }
+            ))
+         await setMatchesInformationListDto(returnedMatchesInformationListDto);
+      }
+
+      async function setTeamsListDtoState(): Promise<void> {
+         if (!matchesListItems) return;
+         const teamsListDto = await getTeamsListDto(tournamentId)
+         await setTeamsListDto(teamsListDto);
+      }
+
+      setTeamsListDtoState().then();
+      setMatchesInformationListDtoState().then();
+
+   }, [matchesListItems]);
+
+   useEffect(() => {
+      async function setPlayersProfilesListDto(): Promise<void> {
+         if (!teamsListDto) return;
+
+         const tournamentPlayersIds = teamsListDto.items
+            .map(({firstTeamPlayer, secondTeamPlayer}) =>
+               [firstTeamPlayer, secondTeamPlayer]
+            ).reduce((acc, teamPlayers) => acc.concat(teamPlayers))
+
+         const x = await Promise.all(tournamentPlayersIds.map((item) => getPlayerProfileDto(item)));
```

Comment:
Nazwa x dla zmiennej to raczej niezbyt dobry pomysł :)

Reply:
Nazwa miala być "xD"
A tak serio:
xd Uciekło mi!!!!
Dzięki za uwagę!